jac-uk/apply
# scripts formatting env:script

## What's included?
Scripts in `package.json` now follow `[ENV]:[SCRIPT]` format and are ordered for easier readability.

## Who should test?
✅ Developers

## How to test?
Observe `package.json` and ensure scripts are correctly formatted and named, ordered and not repeated.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:OFF
_can be OFF, DEVELOP or STAGING_
